Remotecfg reserved system attributes (#1282)
grafana/alloy: docs/sources/reference/config-blocks/remotecfg.md

The `id` and `attributes` fields are used in the periodic request sent to the
remote endpoint so that the API can decide what configuration to serve.

> The `attribute` map keys can include any custom value with the exception of the reserved prefix `collector.`.

Reviewer: ```suggestion
The `attribute` map keys can include any custom value except the reserved prefix `collector.`.
```

> This reserved label prefix is used for automatic system attributes which can't be overridden.

Reviewer: ```suggestion
The reserved label prefix is for automatic system attributes.
You can't override this prefix.
```

> This reserved label prefix is used for automatic system attributes which cannot be overridden.

Reviewer: ```suggestion
This reserved label prefix is used for automatic system attributes which can't be overridden.
```

Reviewer: Approving for docs

> The `attribute` map keys can include any custom value with the exception of a
> reserved prefix `collector_system_attribute_`. This reserved label prefix is
> used for automatic system attributes which cannot be overridden.
> * `collector_system_attribute_os`: The operating system where alloy is running.
> * `collector_system_attribute_version`: The version of alloy.

Reviewer: ```suggestion
The `attribute` map keys can include any custom value with the exception of the reserved prefix `collector_system_attribute_`.
This reserved label prefix is used for automatic system attributes which cannot be overridden.

* `collector_system_attribute_os`: The operating system where {{< param "PRODUCT_NAME" >}} is running.
* `collector_system_attribute_version`: The version of {{< param "PRODUCT_NAME" >}}.
```
